.github/issue_template.md

```diff
@@ -0,0 +1,5 @@
+### Expected Behavior
```

Comment:
Should we add a title to it?

# Issue Description/Title (Emoticon to express our reaction to finding the issue)